beam/core/src/test/java/cz/o2/proxima/beam/direct/io/DirectBatchUnboundedSourceTest.java

```diff
@@ -94,12 +107,13 @@ public void testThroughputParsing() {
     assertEquals(1000, maxThroughput);
   }
 
-  @Test(timeout = 30000)
-  public void testDirectBatchUnboundedSourceWithManyAndThrottling() {
+  @Test(timeout = 30_000)
+  public void testDirectBatchUnboundedSourceWithManyAndThrottling() throws InterruptedException {
+    // Test data size is about 4kb. Let's limit to troughput to 2kb.
     Config config =
         ConfigFactory.parseString(
                 "beam.unbounded-batch.limit.uri = \"inmem:///proxima_gateway\"\n"
-                    + "beam.unbounded-batch.limit.throughput = 10000")
+                    + "beam.unbounded-batch.limit.throughput = 2048")
```

Comment:
This test is known to be flaky. Will look into that, please leave it as is for now.